assets/training/finetune_acft_hf_nlp/components/preprocess/chat_completion/spec.yaml

```diff
@@ -1,14 +1,14 @@
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
 name: chat_completion_datapreprocess
-version: 0.0.53
+version: 0.0.54
 type: command
 
 is_deterministic: True
 
 display_name: Chat Completion DataPreProcess
 description: Component to preprocess data for chat completion task. See [docs](https://aka.ms/azureml/components/chat_completion_datapreprocess) to learn more.
 
-environment: azureml://registries/azureml/environments/acft-hf-nlp-gpu/versions/63
+environment: azureml://registries/azureml/environments/acft-hf-nlp-gpu/versions/64
```

Comment:
Can you update nlp_multiclass nlp_multilabel nlp_ner as well?
those are referring to
environment: azureml://registries/azureml/environments/acft-hf-nlp-gpu/versions/16

Reply:
@msp8955,  we are not owner of the components and do not know how to test them